diff --git a/part1/zmqrpc.py b/part1/zmqrpc.py
index a3580cf..3b9a558 100644
--- a/part1/zmqrpc.py
+++ b/part1/zmqrpc.py
@@ -22,6 +22,7 @@ class Server(threading.Thread):
   def run(self):
     while True:
       req = self.socket.recv_pyobj()
+      print("s<<",req)
       rep = None
 
       try:
@@ -31,6 +32,7 @@ class Server(threading.Thread):
         print(e)
         pass
 
+      print("s>>",rep)
       self.socket.send_pyobj(rep)
 
 class Client:
@@ -54,11 +56,13 @@ class Client:
         self._needs_recv = False
 
     if not self._needs_recv:
+      print(">>",obj)
       self._socket.send_pyobj(obj)
       self._needs_recv = True
 
   def recv_pyobj(self):
     o = self._socket.recv_pyobj()
+    print("<<",o)
     self._needs_recv = False
     return o
 
